STORE_DESCRIPTION.md

```diff
@@ -0,0 +1,251 @@
+# Device Simulator Express, a Microsoft Garage project
+
+Make without limit! Device Simulator Express, a Microsoft Garage project, allows you to code microcontrollers without the hardware on hand! You can program your Circuit Playground Express (CPX) or your BBC Micro:bit! Test and debug your code on the device simulator and see the same
+result when you plug in your actual microcontroller. Curious about the output of the device, the serial
+monitor allows you to observe the device output.
+
+<img alt='CircuitPlayground Express' src=https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/cpx.jpg>
+
+## Build Status
+
+| Branch  |                                                                                                                Build Status                                                                                                                 |
+| :------ | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| dev     |    [![Build Status](https://microsoftgarage.visualstudio.com/Intern%20GitHub/_apis/build/status/Adafruit/Pacifica-CI?branchName=dev)](https://microsoftgarage.visualstudio.com/Intern%20GitHub/_build/latest?definitionId=304&branchName=dev)    |
+| staging |   [![Build Status](https://microsoftgarage.visualstudio.com/Intern%20GitHub/_apis/build/status/Adafruit/Pacifica-CI?branchName=staging)](https://microsoftgarage.visualstudio.com/Intern%20GitHub/_build/latest?definitionId=304&branchName=staging)    |
+| master  | [![Build Status](https://microsoftgarage.visualstudio.com/Intern%20GitHub/_apis/build/status/Adafruit/Pacifica-CI?branchName=master)](https://microsoftgarage.visualstudio.com/Intern%20GitHub/_build/latest?definitionId=304&branchName=master) |

```

Comment:
Is this file supposed to be used in the marketplace? If yes, then I would probably remove the build status. The current status of dev/staging doesn't matter for the version that is shipped to the marketplace.